.travis.yml

```diff
@@ -15,6 +15,8 @@ matrix:
   include:
     - rvm: 2.4.0
       env: COMMAND=rubocop
+before_install:
+  - gem install bundler:1.17.3
```

Comment:
Perhaps grab the bundler version from the `Gemfile.lock`, e.g.  `gem install bundler:"$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)"`

Reply:
Removed this, was sorted in https://github.com/zendesk/zendesk_apps_support/pull/269